django_cprofile_middleware/middleware.py

```diff
@@ -27,12 +30,16 @@ class ProfilerMiddleware(object):
 
     ?count => The number of rows to display. Default is 100.
 
+    ?download => Download profile file suitable for visualization. For example in
+        snakeviz or RunSnakeRun
+
     This is adapted from an example found here:
     http://www.slideshare.net/zeeg/django-con-high-performance-django-presentation.
     """
     def can(self, request):
-        return settings.DEBUG and 'prof' in request.GET and \
-            request.user is not None and request.user.is_staff
+        return ((settings.DEBUG or
```

Comment:
Hmm, it looks like this would enable the middleware on production if the user `is_staff`. I didn't really intend for it to be this way, because it feels like a little bit of a risk to be able to profile code that's running on prod.